src/vivliostyle/table.js

```diff
@@ -172,39 +176,65 @@ goog.scope(function() {
+            var f = function() {
+                var cellFragments = this.getCellFragments();
+                return cellFragments.map(function(cellFragment) {
+                    return cellFragment.findAcceptableBreakPosition();
+                });
+            }.bind(this);
+            this.acceptableCellBreakPositions =
+              repetitiveElements ? repetitiveElements.preventStatusUpdate(f) : f();
         }
         return this.acceptableCellBreakPositions;
     };
 
     /**
+     * @private
+     * @return {number}
+     */
+    BetweenTableRowBreakPosition.prototype.getRowIndex = function() {
+        if (this.rowIndex != null) return this.rowIndex;
+        return this.rowIndex = this.formattingContext.findRowIndexBySourceNode(this.position.sourceNode);
+    };
+
+    /**
+     * @private
+     */
+    BetweenTableRowBreakPosition.prototype.getCellFragments = function() {
+        return this.formattingContext.getRowSpanningCellsOverflowingTheRow(this.getRowIndex()).map(
+                this.formattingContext.getCellFragmentOfCell, this.formattingContext);
+    };
+
+    /**
      * @param {number} rowIndex
      * @param {!adapt.vtree.NodeContext} beforeNodeContext
      * @param {!vivliostyle.table.TableFormattingContext} formattingContext
      * @constructor
-     * @implements {adapt.layout.BreakPosition}
+     * @extends {adapt.layout.AbstractBreakPosition}
      */
-    vivliostyle.table.InsideTableRowBreakPosition = function(rowIndex, beforeNodeContext, formattingContext) {
+    vivliostyle.table.InsideTableRowBreakPosition = function(
+        rowIndex, beforeNodeContext, formattingContext) {
+        adapt.layout.AbstractBreakPosition.call(this);
         /** @const */ this.rowIndex = rowIndex;
         /** @const */ this.beforeNodeContext = beforeNodeContext;
         /** @const */ this.formattingContext = formattingContext;
         /** Array<!adapt.layout.BreakPositionAndNodeContext> */ this.acceptableCellBreakPositions = null;
     };
     /** @const */ var InsideTableRowBreakPosition = vivliostyle.table.InsideTableRowBreakPosition;
+    goog.inherits(InsideTableRowBreakPosition, adapt.layout.AbstractBreakPosition);
 
     /**
      * @override
      */
     InsideTableRowBreakPosition.prototype.findAcceptableBreak = function(column, penalty) {
+        var repetitiveElements =  this.getRepetitiveElements();
+        if (repetitiveElements) repetitiveElements.updateState(penalty);
         if (penalty < this.getMinBreakPenalty())
             return null;
-        var allCellsBreakable = this.getAcceptableCellBreakPositions().every(function(bp) {
-            return !!bp.nodeContext;
+        var cellFragments = this.getCellFragments();
+        var allCellsBreakable = this.getAcceptableCellBreakPositions().every(function(bp, index) {
+            return !!bp.nodeContext && !cellFragments[index].pseudoColumn.isStartNodeContext(bp.nodeContext);
```

Comment:
Fixed it in [this commit](https://github.com/vivliostyle/vivliostyle.js/commit/482e9ff526e08771f3c2da127abca9011686d80b).